render/container_test.go

```diff
@@ -61,7 +61,7 @@ func TestContainerFilterRenderer(t *testing.T) {
 	input.Container.Nodes[fixture.ClientContainerNodeID] = input.Container.Nodes[fixture.ClientContainerNodeID].WithLatests(map[string]string{
 		docker.LabelPrefix + "works.weave.role": "system",
 	})
-	have := Prune(render.ContainerRenderer.Render(input, render.FilterSystem))
```

Comment:
As an aside, we should rename `ContainerWithImageNameRenderer` to just `ContainerRenderer`, as its the only one we use.